lib/active_record/sql_analyzer/background_processor.rb

```diff
@@ -19,7 +19,7 @@ def <<(event)
       def process_queue
         event = @queue.pop
 
-        event[:caller] = SqlAnalyzer.config[:backtrace_filter_proc].call(event[:caller])
+        event[:caller] = event[:caller].map(&SqlAnalyzer.config[:backtrace_filter_proc])
```

Comment:
And I'm not sure why it's being changed... doesn't seem to relate to transactions?